### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by grst**
I tried summarizing the most important points from the google doc into this README. 

Feel free to add more mandatory or optional requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+-   The package should provide tutorials (or "vignettes") as part of the
+    documentation.
+-   The package uses our [cookiecutter
+    template](https://github.com/scverse/cookiecutter-scverse).
+
+Note that the ecosystem packages are not necessarily constrained to Python
+software, as long as they fulfil the above requirements.
+
+If you cannot or do not want to comply with these requirements, you are still
+free to use be interoperable with scverse packages by using our datastructures,
```
Comment (mikelkou) [inline README.md:35]: Instead of "free to use be interoperable" --> "free to be interoperable"

Comment (mikelkou) [inline README.md:35]:
```suggestion
free to be interoperable with scverse packages by using our data structures,
```

**Review by mikelkou — CHANGES_REQUESTED** [review]

**Review by ivirshup — COMMENTED** [review]
Would a section of goals and non-goals be good here too?

Goals including: promotion of ecosystem packages, organized view of packages, things we can test new releases against

Non-goals: in-depth package review and endorsement?

**Comment by grst** [comment]
To avoid redundancies, I think it's better to get rid of the PR template and just ask people to copy the checklist from the README. 
I also don't think it's necessary to ask for the links in the questionnaire as I did before, as these are already part of the `meta.yaml` file.

**Comment by grst** [comment]
> Would a section of goals and non-goals be good here too?

Added a few sentences at the top of the README

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Scverse Ecosystem Packages
+
+This repository contains a list of approved software that works together with
```
Comment (adamgayoso) [inline README.md:3]:
```suggestion
This repository contains a list of verified software that works together with
```
?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+scverse core packages and are listed on our website. The goal is to increase
```
Comment (adamgayoso) [inline README.md:4]:
```suggestion
scverse core packages and are listed on the scverse website. The goal is to increase
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+-   Please refer to other entries for examples
+-   The full definition of available fields is available in [`schema.json`](schema.json)
+-   Please copy the checklist from below into the pull request description and answer all questions.
+
+## What are the requirements for an ecosystem package?
+
+For a package to become an approved ecosystem package, it must fulfill all mandatory requirements from the checklist
+below.
+
+Ecosystem packages are not necessarily constrained to Python software as long as they fulfil the above requirements.
```
Comment (adamgayoso) [inline README.md:29]:
```suggestion
Ecosystem packages can be written in non-Python languages as long as they fulfill the above requirements.
```

**Review by adamgayoso — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ivirshup) [reply README.md:3]: Tbh:

```suggestion
This repository contains a list of software that works together with
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# Scverse Ecosystem Packages
+
+This repository contains a list of approved software that works together with
+scverse core packages and are listed on our website. The goal is to increase
+visibility of ecosystem packages and make it easier for users to find
+appropriate software. In the future, we may also test releases of core packages against
+the test suites of ecosystem packages.
```
Comment (ivirshup) [inline README.md:7]: Only the first sentence changed. Just made each sentence it's own line for ease of editing.

```suggestion
This repository contains the list of scverse ecosystem packages that are displayed on scverse.org.
The goal is to increase visibility of ecosystem packages and make it easier for users to find appropriate software. 
In the future, we may also test releases of core packages against the test suites of ecosystem packages.
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+## What are the requirements for an ecosystem package?
+
+For a package to become an approved ecosystem package, it must fulfill all mandatory requirements from the checklist
+below.
+
+Ecosystem packages can be written in non-Python languages as long as they fulfill the above requirements.
+
+If you cannot or do not want to comply with these requirements, you are still
+free to make your package interoperable with scverse by using our datastructures,
+but we will not list your package on our ecosystem page.
```
Comment (ivirshup) [inline README.md:31]:
```suggestion
If a package is part of this list, it means it fulfills certain minimum requirements as outlined below.
It **does not** mean we endorse the package or performed an in-depth review.

**Hint:** If you want to receive notifications about new ecosystem packages, simply use GitHub's "watch" functionality for this repository.

## How can my package become part of the list?

Submit a pull-request adding a `meta.yaml` file for your package to the `packages` directory.

-   Please refer to other entries for examples
-   The full definition of available fields is available in [`schema.json`](schema.json)
-   Please copy the checklist from below into the pull request description and answer all questions.

## What are the requirements for an ecosystem package?

For a package to become an approved ecosystem package, it must fulfill all mandatory requirements from the checklist below.

Ecosystem packages can be written in non-Python languages as long as they fulfill the above requirements.

If you cannot or do not want to comply with these requirements, you are still free to make your package interoperable with scverse by using our datastructures, but we will not list your package on our ecosystem page.
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+For a package to become an approved ecosystem package, it must fulfill all mandatory requirements from the checklist below.
+
+Ecosystem packages can be written in non-Python languages as long as they fulfill the above requirements.
+
+If you cannot or do not want to comply with these requirements, you are still free to make your package interoperable with scverse by using our datastructures, but we will not list your package on our ecosystem page.
+
+## Checklist for adding packages
+
+### Mandatory
```
Comment (ivirshup) [inline README.md:30]: Why is this section separate from the requirements one above?

Also, I don't think it needs to be a checklist unless it's in the PR template.

File: .gitignore [hunk 8]
```diff
@@ -0,0 +1 @@
+.vscode
```
Comment (ivirshup) [inline .gitignore:1]:
```suggestion
.vscode
.DS_STORE
```

**Review by ivirshup — COMMENTED** [review]
Looking good! I'd be happy with merging soon and doing follow up PRs.

File: README.md (hunk 7 above)
Comment (grst) [reply README.md:30]: The details are gone now from the above section... any chance you've been looking at an old version? 
But we can also make it a subheading...

As I mentioned in the other PR, I didn't like the PR template too much - I think the full checklist should be more accessible than being hidden in a PR template. 

We can wrap this up tonight.

File: README.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+Name of the tool: XXX
+
+Short description: XXX
+
+How does the package use scverse data structures (please describe in a few sentences): XXX
+
+-   [ ] The code is publicly available under an OSI-approved license
```
Comment (adamgayoso) [inline README.md:38]: add hyperlink

File: README.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+-   [ ] The package provides versioned releases
+-   [ ] The package can be installed from a standard registry (e.g. PyPI or conda-forge)
```
Comment (adamgayoso) [inline README.md:40]:
```suggestion
-   [ ] The package can be installed from a standard registry (e.g. PyPI, conda-forge, bioconda)
```

**Review by adamgayoso — CHANGES_REQUESTED** [review]

File: README.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+-   [ ] The package uses automated software tests and continuous integration
+-   [ ] The package provides API documentation
```
Comment (adamgayoso) [inline README.md:42]:
```suggestion
-   [ ] The package provides API documentation via a website or README
```

File: README.md (hunk 9 above)
Comment (adamgayoso) [inline README.md:38]:
```suggestion
-   [ ] The code is publicly available under an [OSI-approved](https://opensource.org/licenses/alphabetical) license
```

**Review by adamgayoso — CHANGES_REQUESTED** [review]

**Pull request merged**